shortcuts.sh

```diff
@@ -132,19 +129,19 @@ function swgjs_create_amp_release_pr() {
 
   __swgjs_get_github_username
 
-  SWG_VERSION=$1
-  if [[ -z "$SWG_VERSION" ]]; then
-    echo "What SwG Version are you releasing? (ex: 0.1.22.333)"
-    read SWG_VERSION
-  fi
-
   swgjs_install
   __swgjs_install_amp
 
-  # Build Swgjs for AMP.
-  cd $SWGJS_PATH
+  # Switch to clean AMP branch.
   git fetch team
-  git checkout team/main
+  git checkout team/amp
+  if [[ `git status --porcelain` ]]; then
+    # Avoid building with local changes.
```

Comment:
Show an error message?

Reply:
By this point, Git already showed a pretty descriptive error message

Reply:
When does git show an error message? git st --porcelain does now show an error if there are local changes.

Reply:
Sorry about that. During testing, I was seeing an error message from Git talking about local changes. I'm not sure what was triggering that. Anyways, I'll create a new PR adding an error message. Nice catch on this!

Reply:
Here's the error message I'm seeing from Git _sometimes_
![image](https://user-images.githubusercontent.com/1216762/203141554-585e3a7f-b3fc-4156-a8bd-81d51edfdf50.png)

Reply:
Anyways, I just added an error message here: https://github.com/subscriptions-project/swg-js/pull/2488